src/dtgtk/paint.c

```diff
@@ -444,6 +444,7 @@ void dtgtk_cairo_paint_eye(cairo_t *cr, gint x, gint y, gint w, gint h, gint fla
 {
   PREAMBLE(1, 0, 0)
 
+  cairo_set_line_width(cr, 0.1);
```

Comment:
line width should not be used anymore. It it set in the `PREAMBLE` macro in a consistent way for each icon.